homeassistant/components/homekit/manifest.json

```diff
@@ -10,6 +10,7 @@
   "loggers": ["pyhap"],
   "requirements": [
     "HAP-python==4.9.1",
+    "chacha20poly1305-reuseable==0.13.2",
```

Comment:
This doesn't look correct. We shouldn't require 3rd party dependencies that are only required by other 3rd party dependencies, and not by Home Assistant itself.

Reply:
The issue here is that `chacha20poly1305-reuseable` generally needs to match to the `cryptography` version which we pin in HA otherwise we need to pin `cryptography` in `chacha20poly1305-reuseable` which would prevent core from updating.

Reply:
`chacha20poly1305-reuseable` is also used in `aioesphomeapi`, `aiohomekit`, `pyatv` which means core installs will have the same problem for those as well since there is nothing force it to update when they update their core install.

Reply:
We could also take the position that its the user's problem for core installs and they need to figure it out (ie revert this PR)

Reply:
Yes, the latter, this isn't a new kind of problem. Users of Home Assistant Core installation type have always been responsible for handling dependency conflicts or other special needs around dependencies.

Reply:
revert https://github.com/home-assistant/core/pull/123505

Reply:
I have never encountered any dependency problems like this before for the last few years for the core installation. This is the first time that the dependency breaks.